### PR TITLE
removed forced slash when using autocomplete

### LINKED ISSUE #838: Windows autocomplete broken by #773
Reported on https://github.com/NVIDIA/DIGITS/issues/47#issuecomment-225852787

@Lucaszw do you mind having a look? Thanks!

### PR DESCRIPTION
**Pull request by Lucaszw**
*Fix #838*
*Comes from #773*

@IsaacYangSLA Would you be able to try this out for me on windows? Thanks!

### CODE REVIEW

File: digits/templates/layout.html [hunk 1]
```diff
@@ -77,15 +77,8 @@
...
-    });
-    $(".autocomplete_path").removeAttr("autocomplete");
-    $(".autocomplete_path").on('keyup',function(e){
-      // If path is relative, enforce absolute
-      var isRelative = e.target.value[0] != '/' && e.target.value[0] != '\\';
-      if (e.target.value.length > 0 && isRelative) {
-        e.target.value = "/" + e.target.value;
-      }
+        },
+        minChars: 0
```
Comment (gheinrich) [inline digits/templates/layout.html]: why did you remove `$(".autocomplete_path").removeAttr("autocomplete");`?

Comment (Lucaszw) [inline digits/templates/layout.html]: My apologies! That was a slip-up on my part (the line is re-added now).

Comment (IsaacYangSLA) [inline digits/templates/layout.html]: The fix seems not working well.  If I type /working/, I could see a list of subfolders and files under c:/working/  (my digits is installed on drive c:).
However, if I type c:/working/, there is no list shown.  At the same time, browser showed two requests, path?query=c and path?query=c%3A.  Nothing else.

Comment (Lucaszw) [inline digits/templates/layout.html]: Interesting... I also changed the minChars count in addition to removing the problematic line for @JVR32. @IsaacYangSLA  Would you be able to check for me if removing `minChars: 0`  fixes the problem for you? Thanks!

**Comment by IsaacYangSLA** [comment]
The autocomplete feature seems different in Windows.
If I type E:\, I get a list of folders/files under E drive.  If I type e:\, C:\ or c:/, nothing.
Replacing \ with /, i.e. E:/, e:/, C:/, c:/  all show nothing.

If I type E:\ and select one folder (data_e), the string changes to E:/data_e, and no autocomplete.

**Comment by Lucaszw** [comment]
All suggested paths should return absolute now, so this should hopefully fix the issue. @IsaacYangSLA Do you mind checking this once more and see if you have any better luck? Thanks

**Comment by IsaacYangSLA** [comment]
@Lucaszw , this is what I saw on Windows after applying e9f907e.
image

C:/, C:\, c:/, c:\ does not return autocomplete list.

The following is shown when I do not type anything.
image

However, if I clicked one folder, the autocomplete didn't show anything inside that folder.

**Comment by Lucaszw** [comment]
Thanks @IsaacYangSLA . Could you try for me selecting an option with the arrow keys and entering a forward slash and seeing if it returns anything?

**Comment by IsaacYangSLA** [comment]
With 0e39e4f, after I clicked one folder, nothing happened.  However, after I type '/' at the end of the folder name, the console immediately showed the list, and the html page showed the same list.

image


The following is the initial state.  Sorry the console showed the list two times.  It's my fault.

image

File: digits/templates/layout.html [hunk 2]
```diff
@@ -71,22 +71,17 @@
        serviceUrl: '/autocomplete/path',
        formatResult: function (suggestion, currentValue)
        {
+         console.log(suggestion);
```
Comment (lukeyeager) [inline digits/templates/layout.html]: Please remove debug statement

**Comment by lukeyeager** [comment]
> With 0e39e4f, after I clicked one folder, nothing happened. However, after I type '/' at the end of the folder name, the console immediately showed the list, and the html page showed the same list.

Yeah, that's the same old behavior we've always had.

**Comment by Lucaszw** [comment]
Alright I think things are good to go. The debug statement has been removed.

**Comment by lukeyeager** [comment]
LGTM!
